src/main/steps/applicant2Sequence.ts

```diff
@@ -10,4 +16,13 @@ export const applicant2Sequence: Step[] = [
     url: HAS_RELATIONSHIP_BROKEN_APPLICANT2,
     getNextStep: () => HOME_URL,
   },
+  {
+    url: HOW_THE_COURTS_WILL_CONTACT_YOU_APPLICANT2,
+    getNextStep: () => ADDRESS_PRIVATE_APPLICANT2,
+  },
+  {
+    url: ADDRESS_PRIVATE_APPLICANT2,
+    // TODO - to be replaced by Enter your postal address when developed as part of NFDIV-757
+    getNextStep: () => HOME_URL,
```

Comment:
I think it probably best to link it to the page it should go to. This way it will satisfy the ACs (even if it shows a 404) and we can have a functional test that checks goes to the correct page e.g. `Then the page URL should be "/applicant2/enter-your-address"`.

```suggestion
    getNextStep: () => YOUR_NAME,
```